servicenow/servicenow.js

```diff
@@ -0,0 +1,164 @@
+//Copyright 2020 Google LLC
```

Comment:
License header is like:
 /**
...
*/
You may follow the format as: https://github.com/GoogleCloudPlatform/dialogflow-integrations/blob/master/spark/server.js

Reply:
corrected header comment